flink-kubernetes/src/test/java/org/apache/flink/kubernetes/artifact/DefaultKubernetesArtifactUploaderTest.java

```diff
@@ -42,6 +44,7 @@
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 /** Tests for {@link DefaultKubernetesArtifactUploader}. */
+@Disabled
```

Comment:
I guess we do not want to disable.

Reply:
yep, artifact that should be removed
done